src/ReinforcementLearningFarm/src/hooks/total_reward_per_episode_last_n.jl

```diff
@@ -0,0 +1,51 @@
+using ReinforcementLearningCore
+using ReinforcementLearningBase
+import Base.push!
+import Base.getindex
+using CircularArrayBuffers: CircularVectorBuffer
+
+"""
+    TotalRewardPerEpisodeLastN{F}(; max_steps = 100)
+
+A hook that keeps track of the total reward per episode for the last `max_steps` episodes.
```

Comment:
Using the wording "steps" might be confusing. Better call it `max_episodes`.

Reply:
Also, would the name `TotalRewardPerLastNEpisodes` make the purpose of the hook more immediatelly obvious?